cordova-lib/package.json

```diff
@@ -22,20 +22,19 @@
     "cordova-js": "4.0.0",
     "cordova-registry-mapper": "1.x",
     "cordova-serve": "^0.1.2",
-    "d8": "0.4.4",
     "dep-graph": "1.1.0",
     "elementtree": "0.1.6",
     "glob": "4.0.6",
     "init-package-json": "^1.2.0",
-    "npm": "1.3.4",
-    "npmconf": "0.1.16",
+    "npm": "^2.10.x",
```

Comment:
Yes. I hope that author of npm and semver will follow own versioning startegy, so minor version changes won't break anything.